tests/operator_test.go

```diff
@@ -2229,6 +2239,7 @@ spec:
 	Context("With PrometheusRule Enabled", func() {
 
 		BeforeEach(func() {
+			tests.SkipIfARM64("Currently not works on Arm64")
```

Comment:
Could you rephrase these:

```suggestion
			tests.SkipIfARM64("Currently not supported on arm64")
```